pkg/dashboard/core/workflow.go

```diff
@@ -51,15 +51,16 @@ const (
 
 // WorkflowMeta defines the root structure of a workflow.
 type WorkflowMeta struct {
-	ID        uint           `gorm:"primary_key" json:"id"`
-	UID       string         `gorm:"index:workflow_uid" json:"uid"`
-	Namespace string         `json:"namespace"`
-	Name      string         `json:"name"`
-	Entry     string         `json:"entry"` // the entry node name
-	CreatedAt time.Time      `json:"created_at"`
-	EndTime   string         `json:"end_time"`
-	Status    WorkflowStatus `json:"status,omitempty"`
-	Archived  bool           `json:"-"`
+	ID         uint           `gorm:"primary_key" json:"id"`
+	UID        string         `gorm:"index:workflow_uid" json:"uid"`
+	Namespace  string         `json:"namespace"`
+	Name       string         `json:"name"`
+	Entry      string         `json:"entry"` // the entry node name
+	CreatedAt  time.Time      `json:"created_at"`
+	EndTime    string         `json:"end_time"`
+	FinishTime time.Time      `json:"finish_time"`
```

Comment:
You can use `EndTime` instead of `FinishTime`. In addition, I don't find where the logic to set `FinishTime`.

Reply:
@STRRL PTAL

Reply:
@cwen0 thanks for your feedback. I will go with `EndTime` and to set it, I need to add logic to ` MarkAsArchived` and `MarkAsArchivedWithUID` methods in the `workflow` store, right?

Reply:
Please use `EndTime` instead of `FinishTime` within workflow.

Reply:
@rksharma95 You can just remove `FinishTime` and use `EndTime` directly.